temporalio/bridge/worker.py

```diff
@@ -0,0 +1,101 @@
+"""Worker using SDK Core."""
+
+from dataclasses import dataclass
+from typing import Iterable
+
+import temporal_sdk_bridge
+from temporal_sdk_bridge import PollShutdownError
+
+import temporalio.api.common.v1
+import temporalio.bridge.client
+import temporalio.bridge.proto
+import temporalio.bridge.proto.activity_task
+import temporalio.bridge.proto.common
+import temporalio.bridge.proto.workflow_activation
+import temporalio.bridge.proto.workflow_completion
+
+
+@dataclass
+class WorkerConfig:
+    """Python representation of the Rust struct for configuring a worker."""
+
+    namespace: str
+    task_queue: str
+    max_cached_workflows: int
+    max_outstanding_workflow_tasks: int
+    max_outstanding_activities: int
+    max_outstanding_local_activities: int
+    max_concurrent_wft_polls: int
+    nonsticky_to_sticky_poll_ratio: float
+    max_concurrent_at_polls: int
+    no_remote_activities: bool
+    sticky_queue_schedule_to_start_timeout_millis: int
+    max_heartbeat_throttle_interval_millis: int
+    default_heartbeat_throttle_interval_millis: int
+
+
+class Worker:
```

Comment:
Not really because it's in a the "bridge" package that no customer-facing code sees/uses. Technically I suppose when you start typing "Wo" in your IDE all things called worker from all packages appear (same for "Client" and similar). I could prefix this with an underscore if this was a concern.